tests/integration/tier1/method/activation_key.py

```diff
@@ -2,6 +2,7 @@
 
 
 def test_activation_key_conversion(convert2rhel):
+    # noinspection PyRedundantParentheses
     with convert2rhel(
         ("-y --no-rpm-va --serverurl {} -k {} -o {} --debug").format(
             env.str("RHSM_SERVER_URL"),
```

Comment:
Since we are changing a few things, what do you think about removing this comment and the parenthesis as well?

Reply:
Good point, I was thinking of removing the parenthesis for a while, guess we can do it now. Thank you!